Woodoku/src/Entity/WoodokuBoard.py

```diff
@@ -75,6 +77,7 @@ def is_not_occupied(self, blocks_coord: List[Tuple[int, int]]) -> bool:
                 return False
         return True
 
+    @staticmethod
     def __validate(block: Tuple[int, int]) -> None:
```

Comment:
why the static method? these methods are meant to me private no?

Reply:
Agree. According to my understanding, this method decides whether the position is valid or not on the board associated with the instance.

Reply:
nvm, I got it.

but it shouldn't be fixed with static method decorators.
should just add self to the parameters to fix my bugs 😅